OrbitClientModel/CaptureSerializerTest.cpp

```diff
@@ -129,6 +148,20 @@ TEST(CaptureSerializer, GenerateCaptureInfo) {
   EXPECT_EQ(callstack_event.time(), actual_callstack_event.time());
   EXPECT_EQ(callstack_event.callstack_hash(), actual_callstack_event.callstack_hash());
 
+  ASSERT_EQ(1, capture_info.tracepoint_infos_size());
+  const orbit_client_protos::TracepointInfo& actual_tracepoint_info =
+      capture_info.tracepoint_infos(0);
+  ASSERT_EQ(actual_tracepoint_info.category(), selected_tracepoint_info.category());
+  ASSERT_EQ(actual_tracepoint_info.name(), selected_tracepoint_info.name());
+  ASSERT_EQ(actual_tracepoint_info.tracepoint_info_key(), 0);
+
+  ASSERT_EQ(2, capture_info.tracepoint_event_infos_size());
```

Comment:
Please add a one line comment here with the explanation from the commit message. This is somewhat unexpected here otherwise.

Reply:
Of course! Thank you! I will.